xml_inscriptions/metadata_only/MO.SL.CAM.G.4-1933.xml

```diff
@@ -91,36 +116,40 @@
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
          <change when="2020-04-19" who="Gaia Gianni">Added metadata and external reference</change>
+         <change when="2021-07-06" who="Sophia Papandonatou">updated metadata, added provenance and
+            transcription</change>
       </revisionDesc>
    </teiHeader>
    <!--facsimile element goes here--><text>
       
         <body>
         <div type="edition">
          <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+           <ab>
+              <lb n="1"/><persName><name key="Καλλισθένης Kallisthene">Καλλισ<choice><sic>ο</sic><corr>θ</corr></choice>ένης</name></persName> <persName><name key="Παιανιεύς Paianieus">Παιανιεὺς</name><note>!</note></persName>
```

Comment:
Kallisthenes or Kallisthene?

Reply:
Paianieus' `@type` should be demotic , no?

Reply:
<name key="..." type="Demotic">?